aaad/authorizers.py

```diff
@@ -175,9 +175,11 @@ def filter_response(self, resource, data, actas):
             return framework.filterResponseBody(data, allowed_namespaces, resource)
 
         def _parse_permissions_file(self, filename):
-            with open(filename, 'r') as data_file:
-                return yaml.load(data_file)
-            return ''
+            permissions = {}
+            for item in filename.split(','):
```

Comment:
A trailing comma would mess it up (e.g. `file1.yml,file2.yml,`)  so an `if item.tstrip():` might be useful.  Although if it fails loudly it's probabyl OK.